app/locale/en.js

```diff
@@ -1225,6 +1225,8 @@ module.exports = {
       auth_tab: 'Sign Up',
       inventory_caption: 'Equip your hero',
       choose_hero_caption: 'Choose hero, language',
+      change_langugae_caption: 'Choose language',
+      change_language_tab: 'Code Lang',
```

Comment:
```suggestion
      change_language_tab: 'Code Language',
```

Reply:
we can't. there's no space for full language text 😂

Reply:
Wrap it onto two lines or just say language?